### DISCUSSION TITLE
giscus.app domain is down

### DISCUSSION THREAD

**Discussion by spirillen** [giscus/giscus#1286] [General]
The servers are clearly misconfiguration and the site is not available online, are you using some censorship mechanism to prevent certain democratic loving peoples out? such as Tor users or other that are preventing tracking of their online activities?

![image](https://github.com/giscus/giscus/assets/44526987/4d9f581d-f3d6-4ce7-aa36-e61aaac0827a)

**Comment by laymonage** [comment]
I don’t use any censorship, the domain points to Vercel’s IP directly and the project is deployed automatically with the same code you see on GitHub. This is a known intermittent problem with Vercel deployments (you can search “Vercel China”), and there’s usually nothing I can do to fix it (sometimes it fixes itself).

Please do your research before accusing someone.

**Reply by spirillen** [reply]
> I don’t use any censorship, the domain points to Vercel’s IP

I don’t use any censorship != the domain points to Vercel’s IP

So you are censoring the tor network deliberately... Why not just say it as it is?

**Reply by laymonage** [reply]
I've never used Tor and I have no idea why it can't access Vercel's server. I have no intentions to block Tor network specifically, so if there's anything I can do to fix it please let me know instead of doubling down with the accusations.
